webapp/sources/ldap-inventory/inventory-api/src/main/scala/com/normation/inventory/domain/MemorySize.scala

```diff
@@ -76,7 +76,7 @@ object MemorySize {
    *
    */
   private val num_r = """(\d[\d_ ]*)""".r
-  private val numunit_r = """(\d[\d_ ]*) *([\p{Alpha}]?[B]?)""".r
+  private val numunit_r = """(\d[\d_ ]*) *([\p{Alpha}]{0,2})""".r
```

Comment:
we used to have ?[BOo], i think we ought to stick to that

Reply:
It was a bug. It does not work with just "k", and now it's advertised in the error message.